Structural/Decorator/Webservice.php

```diff
@@ -21,7 +21,7 @@ public function __construct($data)
     }
 
     /**
-     * @return string
+     * @return mixed
```

Comment:
Maybe better tweak type in the interface? I was sure `renderData()` should return a `string` type, wasn't it?